packages/shortest/src/cli/commands/init.ts

```diff
@@ -161,10 +422,14 @@ export const executeInitCommand = async () => {
 
   try {
     await tasks.run();
-    console.log(pc.green("\nInitialization complete! Next steps:"));
-    console.log(`1. Update ${ENV_LOCAL_FILENAME} with your values`);
-    console.log("2. Create your first test file: example.test.ts");
-    console.log("3. Run tests with: npx/pnpm test example.test.ts");
+    if (tasks.ctx.generateSampleTestFile) {
+      console.log(pc.green("\nSetup complete!"));
+      console.log("Run tests with: npx/pnpm shortest");
+    } else {
+      console.log(pc.green("\nInitialization complete! Next steps:"));
+      console.log("1. Create your first test file: example.test.ts");
+      console.log("2. Run tests with: npx/pnpm shortest example.test.ts");
```

Comment:
Technically, we could [detect the package manager](https://github.com/antiwork/shortest/blob/6aad16a405d734dc2b1aa0d4adac2103f1423cdb/packages/shortest/src/cli/commands/init.ts#L452-L454), pass that info around, and craft the exact command that needs to be executed.